blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/Configuration.java

```diff
@@ -65,6 +65,20 @@ public void init() throws IOException {
         return Collections.unmodifiableMap(props);
     }
 
+    public Map<Object, Object> getAllProperties() {
```

Comment:
`getProperties` does NOT return the default values. See `testGetPropertiesDoesntWorkForDefaults`. What it does is treat the `props` object as a `Map`: all calls to `Map.get` will bypass the `Properties.getProperty` method, which is where default values are handled.